### ISSUE TITLE
Can you explain in detail how SAM is obtained for each dataset？

### ISSUE THREAD

**Issue by Aliplore** [CVL-hub/GPRN#2]
Can you explain in detail how SAM is obtained for each dataset？

**Comment by sfengpeng** [comment]
I guess what you mean is how to obtain SAM’s masks for each dataset. 

The masks are generated from SAM using automatic mode. For this mode, we use default hyper-parameters as SAM. Specifically, a number of points (32 x 32 grids) across the input image are sampled and prompted to SAM. Then the generated masks are post-processed as default.

If you have any further questions, please feel free to contact me.

**Comment by CVL-hub** [comment]
I guess what you mean is how to obtain SAM’s masks for each dataset.

The masks are generated from SAM using automatic mode. For this mode, we use default hyper-parameters as SAM. Specifically, a number of points (32 x 32 grids) across the input image are sampled and prompted to SAM. Then the generated masks are post-processed as default.

If you have any further questions, please feel free to contact me.

**Comment by Aliplore** [comment]
Can you provide the training, fine-tuning, and testing parameters for each dataset? You only provided the parameters for the Deepglobe dataset being tested

**Comment by CVL-hub** [comment]
Hi, we have provided these parameters in the run.sh file.

During training, all dataset parameters are the same. For fine-tuning, please refer to the parameters for each dataset in run.sh.

Good luck!
